public/forms/validation/form.ttl

```diff
@@ -0,0 +1,159 @@
+  form:hasFieldGroup ext:validationsFG ;
+  form:includes ext:validationResultMessageF .
+
+ext:validationsFG a form:FieldGroup ;
+  form:hasField fields:validatonSelectorF .
+
+##########################################################
+#  Validation type selector
+##########################################################
+fields:validatonSelectorF a form:Field ;
+  sh:name "Validation type" ;
+  sh:order 2 ;
+  sh:path rdf:type ;
+  form:help """
+   """;
+  form:validations 
+    [ a form:RequiredConstraint ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        sh:resultMessage "Dit veld is verplicht."@nl
+    ] ;
+  form:displayType displayTypes:conceptSchemeSelector ;
+  form:options """{"conceptScheme":"http://lblod.data.gift/concept-schemes/dde3d2a3-e848-47ea-ba44-0f2e565f04ab","searchEnabled":true}""" ;
+  sh:group ext:formFieldPg ;
+  form:hasConditionalFieldGroup fields:maxLengthValidationCFG .
+
+##########################################################
+#  Validation generator
+##########################################################
+ext:addValidationGenerator a form:Generator ;
+  form:prototype [
+    form:shape [
+      form:grouping "Bag"@en
+    ]
+  ] .
+
+##########################################################
+#  Field Groups per validation type
+##########################################################
+fieldGroups:maxLengthValidationFG a form:FieldGroup ;
+  form:hasField ext:validationMaxLengthF .
+
+##########################################################
+#  Validation type conditions
+##########################################################
+fields:maxLengthValidationCFG a form:ConditionalFieldGroup ;
+  form:conditions
+    [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        form:conceptScheme concept:dde3d2a3-e848-47ea-ba44-0f2e565f04ab ;
+        form:customValue form:MaxLength
+    ] ;
+  form:hasFieldGroup fieldGroups:maxLengthValidationFG .
+
+##########################################################
+#  All separate fields
+##########################################################
+    ##########################################################
+    #  Error message field
+    ##########################################################
+    ext:validationResultMessageF a form:Field ;
+      sh:name "Error message" ;
+      sh:order 100 ; # I think we always want to show this below the conditional fields?
+      sh:path sh:resultMessage ;
+      form:displayType displayTypes:defaultInput ;
+      sh:group ext:formFieldPg .
+
+    ##########################################################
+    #  Max-lenght field
+    ##########################################################
+    ext:validationMaxLengthF a form:Field ;
+      sh:name "Max characters" ;
+      sh:order 101 ;
+      sh:path form:max ;
+      form:validations 
+        [ a form:RequiredConstraint ;
+            form:grouping form:Bag ;
+            sh:path rdf:type ;
```

Comment:
```suggestion
            sh:path form:max;
```

It doesn't trigger the validation since the type is always set.

Reply:
Indeed I now see the difference :D
![image](https://github.com/lblod/frontend-form-builder/assets/36266973/d30c6f87-cd96-4b93-8e4a-cc93eb23f894)